jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/card/CardWithGridMarkupTests.java

```diff
@@ -0,0 +1,156 @@
+package io.github.epam.bootstrap.tests.composite.section.card;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static com.epam.jdi.light.elements.common.Alerts.validateAlert;
+import static io.github.com.StaticSite.bsPage;
+import static io.github.com.pages.BootstrapPage.cardWithGridMarkup;
+import static io.github.epam.bootstrap.tests.BaseValidations.baseValidation;
+import static io.github.epam.states.States.shouldBeLoggedIn;
+import static org.hamcrest.Matchers.is;
+
+public class CardWithGridMarkupTests extends TestsInit {
+
+    private String textTitleCardWithGridMarkup11 = "TITLE";
+    private String textTitleCardWithGridMarkup12 = "TITLE";
+    private String textTitleCardWithGridMarkup21 = "TITLE";
+    private String textTitleCardWithGridMarkup22 = "TITLE";
+
+    private String textParagraphCardWithGridMarkup11 = "1st row 1st cell";
+    private String textParagraphCardWithGridMarkup12 = "1st row 2nd cell";
+    private String textParagraphCardWithGridMarkup21 = "2nd row 1st cell";
+    private String textParagraphCardWithGridMarkup22 = "2nd row 2nd cell";
+
+
+    private String textButtonCardWithGridMarkup11 = "Click me";
+    private String textButtonCardWithGridMarkup12 = "Click me";
+    private String textButtonCardWithGridMarkup21 = "Click me";
+    private String textButtonCardWithGridMarkup22 = "Click me";
+
+    private String textAlert = "Button Clicked!";
+
+    @BeforeMethod
+    public void before() {
+        shouldBeLoggedIn();
+        bsPage.shouldBeOpened();
+    }
+
+    @Test
+    public void baseValidationCardWithGridMarkup11Test() {
+        baseValidation(cardWithGridMarkup);
+        baseValidation(cardWithGridMarkup.listCard.get(0));
```

Comment:
we should use 1 as first index in test cases , please rewrite (or wait when new weblist will be ready to work with 1, it will be soon)